models/vrp.rb

```diff
@@ -383,7 +382,8 @@ def self.convert_expected_string_to_symbol(hash)
       }
 
       hash[:services].each{ |service|
-        service[:skills] = service[:skills]&.map(&:to_sym)
+        service[:skills] ||= [] # Service skills should always be initialized
```

Comment:
I can't understand why we need this `service[:skills] ||= []` and `vehicle[:skills] = [[]]`? Is it for skipping the `service.has_key?(:skill)` logic? The initialization of a member should not be obligatory (or handled here), the service class can initialize its own members itself. Am I missing something?

Reply:
I was concerned with the behavior change but I guess this is safe and your explanation is :ok_hand: